gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/ConfigBasedDatasetsFinder.java

```diff
@@ -50,36 +63,43 @@
  */
 
 @Slf4j
-public class ConfigBasedDatasetsFinder implements DatasetsFinder<ConfigBasedDataset> {
+public class ConfigBasedDatasetsFinder implements DatasetsFinder<CopyableDatasetBase> {
+
+  //prefix key for config store used by data replication
+  public static final String GOBBLIN_REPLICATION = "gobblin.replication";
```

Comment:
So far we've used the namespace `gobblin.copy`. I know it's not the ideal name, but it would be good to have consistency.